Floofbot/Modules/Administration.cs

```diff
@@ -106,6 +106,11 @@ public class Administration : ModuleBase<SocketCommandContext>
                 return;
             }
 
+            if(reason.Length > 500) {
```

Comment:
I guess you'll want to warn mods of this ahead of time around time of deployment.

Reply:
agreed

I don't think we have had a warning that was over this in the past